src/main/java/dk/cngroup/lentils/controller/TeamController.java

```diff
@@ -51,14 +52,22 @@ public Error handleNotFoundException(InputMismatchException e)
 
     @GetMapping("/update/{id}")
     public String update(@PathVariable("id") Long id, Model model) {
-        fillModelAttributes(model, teamService.getAll(), teamService.get(id).get(), ACTION_TEAM_UPDATE + id);
+        try {
```

Comment:
tady to try/catch nebude. V kontroleru bude metoda, ktera bude zachytavat vsechny vyjimky. To udela spring za tebe ;) Takze tuto metodu nech tak, jak byla. ;)